tools/packaging/static-build/agent/build.sh

```diff
@@ -16,16 +17,16 @@ source "${script_dir}/../../scripts/lib.sh"
 container_image="${AGENT_CONTAINER_BUILDER:-$(get_agent_image_name)}"
 [ "${CROSS_BUILD}" == "true" ] && container_image="${container_image}-cross-build"
 
-sudo docker pull ${container_image} || \
+sudo docker pull --platform=linux/"${target_arch}" "${container_image}" || \
```

Comment:
@jongwu This fixes the problem with pulling an amd64 image to an arm64 machine. Once this is merged one of the CI jobs will create that arm64 build and we can reuse it.

Reply:
Good, thanks @zvonkok